test/Tizen.NUI.PerformanceTest/PerformanceTest.cs

```diff
@@ -475,7 +488,7 @@ protected override void OnCreate()
     [STAThread] // Forces app to use one thread to access NUI
     static void Main(string[] args)
     {
-        PerfNewExample example = new PerfNewExample();
+        PerformanceTestExample example = new PerformanceTestExample();
         example.Run(args);
     }
 }
```

Comment:
this is trivial thing but from next time, could you add new line at the end of file?
reference: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Fix done. thank you.